liminal-manual/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for liminal-manual.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 673156464838.dkr.ecr.us-west-2.amazonaws.com/liminal-manual
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
I am using `latest` here on purpose, and expect this to be overwritten by `helmfile` or by Keel.

The idea is to point to a "default" image and the tag to be decided by another level of configuration management.

In the case of Keel (outside of gitops model), and given `latest` tag does not exist for this project, it allows to make sure we will never rollback/downgrade to a previous version in case we force ArgoCD to deploy, or if we `helmfile template` and apply the manifests.

Kubernetes rolling update would ensure:
- the update will fail
- the service would not be down given the new pods won't start